ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -125,6 +125,35 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-assisted-operator
+- as: e2e-metal-assisted-operator-bmh
+  steps:
+    cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        NUM_WORKERS=2
+        NUM_EXTRA_WORKERS=1
+        PROVISIONING_NETWORK_PROFILE=Disabled
+    post:
+    - ref: baremetalds-assisted-operator-gather
+    - ref: baremetalds-devscripts-gather
+    - ref: baremetalds-packet-teardown
+    pre:
```

Comment:
maybe replace ``pre`` and ``post`` to make it similar to the actual order of steps